src/public/device.ts

```diff
@@ -0,0 +1,93 @@
+import { GlobalVars } from '../internal/globalVars';
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { frameContexts } from '../internal/constants';
+
+/**
+ * Namespace for device related APIs. These are implemented only for mobile (Android and iOS)
+ * On desktop, these APIs will be a noop.
+ */
+export namespace device {
+  /**
+   * Status codes for device APIs
+   * @see File
+   */
+  export enum StatusCode {
```

Comment:
Can we keep an error object in top level response object ? Like we discussed in API versioning review meeting, also in review here - https://github.com/OfficeDev/microsoft-teams-library-js/pull/326
It also tries to define some standard error codes which can be used across SDK APIs. Would recommend we use same codes here.

Reply:
I changed this to ErrorCode. Yes this is similar to what we are trying to do in versioning. Once that is checked in, we can migrate the ErrorCode to single place. As of now keeping it here. Also I think we won't need an Error object, rather an ErrorCode should suffice to convey the error. App can show appropriate text to the user. Please see my comment in PR #326